rapier/src/lib.rs

```diff
@@ -53,7 +53,7 @@ pub struct RapierPlugin {
     pub parameters: IntegrationParameters,
 }
 
-/// Components automatically register, by the plugin that reference the body in rapier's world
+/// Components automatically register, by the plugin that references the body in rapier's world
 ///
 /// It can be used to get direct access to rapier's world.
```

Comment:
```suggestion
/// Component automatically added to the entities that references the body in rapier's world.
///
/// It can be used to get direct access to rapier's world.
///
/// It can also be used to make heron aware of rapier entities that have been created directly (without using heron's [`Body`](heron_core::Body) component). In this case, it is also important to tell heron what is the type of rigid body by adding a [`BodyType`](heron_core::BodyType).
```